packages/docs/src/pages/guides/merging-themes.mdx

```diff
@@ -59,4 +58,23 @@ export default {
 }
 ```
 
+## Merging theme values from multiple files
+
+For the core scales (`colors`, `fontSizes`, `space`, etc), it’s typically simpler to understand a theme when values are defined in one file.
+However, especially for projects with many [styles](/styling-mdx) or [variants](/guides/variants), you may want to split values that end up
+as part of one object across multiple files. You can use the same kind of imports combined with the `merge` utility for this.
 
+```js
+// example theme/index.js
+import { merge } from 'theme-ui'
+import colors from './colors'
+import fonts from './fonts'
+import textStyles from './styles/text'
+import tableStyles from './styles/tables'
+
```

Comment:
What do you think about adding some more here? I'd like to show that shorthand property names and object spreading _still work_ like with any other object.

```suggestion
import { buttons, forms, links } from './variants'
```

Reply:
Looks great, will implement!